index.html

```diff
@@ -9,6 +9,31 @@
   </head>
   <body class="container">
     <h1>RKI Formulare</h1>
+    <p>Auf dieser Seite kansst du mithilfe der Corona-Warn-App RKI-Formulare automatisch mit deinen Daten ausfüllen. Drücke einfach einen der beiden Knöpfe und scanne dann den QR code von dem Testprofil.</p>
+
+    <details>
+      <summary>Wie erstelle ich mein Corona-Warn-App-Testprofil?</summary>
+      <p>Um ein Schnelltest-Profil anzulegen, können Nutzer*innen auf der Startseite ihrer Corona-Warn-App unter „Test registrieren“ 
+        auf Nächste Schritte gehen. Dort finden Sie den Bereich „Schnelltest-Profil anlegen“.</p>       
+      
+      <div id="spacer">
+        <img id="screenshot" src="schritt1.png" width="296" height="640">
+        <img id="screenshot" src="schritt2_profil_anlegen.png" width="296" height="640">
+      </div>
+      
+      <p>Dort können sie ihre persönlichen Daten, wie Name, Adresse und Telefonnummer, eintragen. Wenn alles eingetragen ist, 
+        können sie die Daten speichern. Die Corona-Warn-App erstellt daraufhin einen QR-Code aus den Daten, den Nutzer*innen an den Teststellen vorzeigen können.</p>
+        
+
+      <p>Einmal angelegt, bleibt das Profil bestehen und kann bei jedem Test wieder vorgezeigt werden. Das Profil ist nur auf dem jeweils eigenen Smartphone in der Corona-Warn-App abgespeichert! Nutzer*innen 
+        können es wieder entfernen, indem sie unter iOS im Schnelltest-Bereich der Corona-Warn-App auf ihr Schnelltest-Profil gehen und dort „Schnelltest-Profil entfernen“ auswählen. Android-Nutzer*innen können 
+        das Profil entfernen, indem sie im Schnelltest-Bereich der Corona-Warn-App auf ihr Schnelltest-Profil gehen und dort im 3-Punkte-Menü "Entfernen" auswählen.</p>
```

Comment:
dieser Text sollte in den nächsten `details`-Block und die beiden `details`-Blöcke sollten einfach untereinander sein, nicht ineinander verschachtelt